SU-HSDO/suhumsci
# Hotfix: Linked cards with incorrect background color

# READY FOR REVIEW

## Summary
- Updates Percy visual regression testing to include the main card collection URL
- Restored the `tertiary-reversed` background color for cards with a description field
- Adds SCSS comments to clarify style changes

## Need Review By (Date)
- asap

## Urgency
medium

## Steps to Test
1. Spin up the https://planning-humsci.suhumsci.loc/ and verify the homepage cards have a grey background:
![Screenshot from 2023-04-03 12-14-42](https://user-images.githubusercontent.com/369018/229580930-631792ff-8acc-4e26-b649-e0b1132eee70.png)
2. Go to go to https://hs-traditional.suhumsci.loc/collections-cards-uniform-and-stretch-1 and verify the grey background is present and the title background is extended if the `hb-stretch-vertical-linked-cards` utility class is present:
![Screenshot from 2023-04-03 08-42-26](https://user-images.githubusercontent.com/369018/229581395-f4a12b87-2f33-4b07-b6be-c57082e37901.png)
3. If the `hb-stretch-vertical-linked-cards` utility class is NOT present the grey background for the description persists but the red background is only applied to the title: 
![Screenshot from 2023-04-03 08-28-49](https://user-images.githubusercontent.com/369018/229581598-36e78b8a-5eb0-4794-bb6a-a35ed0ea10f4.png)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
